docs/guides/cluster-to-advanced-cluster-migration-guide.md

```diff
@@ -0,0 +1,175 @@
+# Cluster to Advanced Cluster Migration Guide
+
+**Objective**: Guide users to replace the `mongodbatlas_cluster` resource with the `mongodbatlas_advanced_cluster` resource.
+
+**Note**: This guide focus on the resource migration as the data source migration is only requiring a resource_type change from `data.mongodbatlas_cluster` to `data.mongodbatlas_advanced_cluster`.  However, pay attention to the [output changes.](#output-changes)
+
+## Main Changes Between `mongodbatlas_cluster` and `mongodbatlas_advanced_cluster`
+
+Key differences in the configuration:
+
+1. Replication Spec Configuration: Supports different node types (electable, analytics, read_only) where hardware configuration can differ between node types.
+2. Provider Settings: Moved from the top level to the replication spec allowing you to create multi-cloud clusters.
+3. Auto Scaling: Moved from the top level to the replication spec allowing you to scale replication specs individually.
+4. Backup Configuration: Renamed from `cloud_backup` to `backup_enabled`.
+5. See also ["Migration to new sharding schema and leveraging Independent Shard Scaling"](advanced-cluster-new-sharding-schema.md#migration-sharded)
+
+### Example1: Old Configuration
+
+```terraform
+resource "mongodbatlas_cluster" "this" {
+    project_id                   = var.project_id
+    name                         = "legacy-cluster"
+    cluster_type                 = "REPLICASET"
+
+    # Provider Settings "block"
+    provider_instance_size_name = "M10" # 1 Replication Spec Configuration
+    provider_name               = "AWS" # 2 Provider Settings
+    
+    auto_scaling_disk_gb_enabled = true # 3 Auto Scaling
+    cloud_backup                 = true # 4 Backup Configuration
+
+    replication_specs {
+     num_shards = 1
+     regions_config {
+       region_name     = "US_EAST_1"
+       priority        = 7
+       electable_nodes = 3 # 1 Replication Spec Configuration
+       analytics_nodes = 1 # 1 Replication Spec Configuration
+       read_only_nodes = 0 # 1 Replication Spec Configuration
+       }
+     }
+
+    }
+```
+
+### Example2: New Configuration
```

Comment:
```suggestion
### Example 2: New Configuration (mongodbatlas_advanced_cluster) 
```

Reply:
Thanks! Changed in 678969f0633a27d16ba9c8891007e4bbf8b1d76d